modules/48_carbonpriceRegi/none/realization.gms

```diff
@@ -0,0 +1,12 @@
+*** |  (C) 2006-2020 Potsdam Institute for Climate Impact Research (PIK)
+*** |  authors, and contributors see CITATION.cff file. This file is part
+*** |  of REMIND and licensed under AGPL-3.0-or-later. Under Section 7 of
+*** |  AGPL-3.0, you are granted additional permissions described in the
+*** |  REMIND License Exception, version 1.0 (see LICENSE file).
+*** |  Contact: remind@pik-potsdam.de
+*** SOF ./modules/48_carbonpriceRegi/none/realization.gms
+
+*####################### R SECTION START (PHASES) ##############################
```

Comment:
I would assume that in this realization the additional taxes are also declared but set to 0